cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-deploying/cloud-experts-getting-started-detailed-cli-guide.adoc

```diff
@@ -0,0 +1,426 @@
+
+* Nodes:
+** 3 control plane nodes
+** 2 infrastructure nodes
+** 2 worker nodes
+** No autoscaling
+** See the documentation on xref:../../../rosa_planning/rosa-sts-aws-prereqs.adoc#rosa-ec2-instances_rosa-sts-aws-prereqs[ec2 instances]for more details.  
+* Region: As configured for the `aws` CLI
+* Networking IP ranges:
+** Machine CIDR: 10.0.0.0/16
+** Service CIDR: 172.30.0.0/16
+** Pod CIDR: 10.128.0.0/14
+* New VPC
+* Default AWS KMS key for encryption
+* The most recent version of OpenShift available to `rosa`
+* A single availability zone
+* Public cluster
+
+=== Checking the installation status
+. Run one of the following commands to check the status of your cluster:
++
+* For a detailed view of the status, run:
++
+[source,terminal]
+----
+rosa describe cluster --cluster <cluster-name>
+----
+* For an abridged view of the status, run:
++
+[source,terminal]
+----
+rosa list clusters
+----
+
+The cluster state will change from “waiting” to “installing” to "ready". This will take about 40 minutes.
+
+. Once the state changes to “ready” your cluster is installed.
+
+== Manual Mode
+If you want to review the roles and policies before applying them to a cluster, use the manual method. This method requires running a few extra commands to create the roles and policies.
+
+In this section we will make use of the `--interactive` mode.  See the documentation on xref:../../../rosa_install_access_delete_clusters/rosa-sts-interactive-mode-reference.adoc#rosa-sts-interactive-mode-reference[interactive mode] for a description of the fields in this section.
+
+=== Creating account roles
+. If this is the _first_ time you are deploying ROSA in this account and you have _not_ yet created the account roles, create the account-wide roles and policies, including the Operator policies. The command creates the needed JSON files for the required roles and policies for your account in the current directory. It also outputs the `aws` CLI commands that you need to run in order to create these objects.
++
+Run the following command to create the needed files and output the additional commands:
++
+[source,terminal]
+----
+rosa create account-roles --mode manual
+----
++
+.Example output
+[source,terminal]
+----
+I: All policy files saved to the current directory
+I: Run the following commands to create the account roles and policies:
+aws iam create-role \
+--role-name ManagedOpenShift-Worker-Role \
+--assume-role-policy-document file://sts_instance_worker_trust_policy.json \
+--tags Key=rosa_openshift_version,Value=4.8 Key=rosa_role_prefix,Value=ManagedOpenShift Key=rosa_role_type,Value=instance_worker
+aws iam put-role-policy \
+--role-name ManagedOpenShift-Worker-Role \
+--policy-name ManagedOpenShift-Worker-Role-Policy \
+--policy-document file://sts_instance_worker_permission_policy.json
+----
+
+. Check the contents of your current directory to see the new files. We will use the `aws` CLI to create each of these objects.
```

Comment:
Should this include "We will" or would just "Use (at beginning of sentence) suffice? I suppose another CLI tool could be used, so that may not work, but not sure if we should use the phrase "We will use"